pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.1</version>
+        <version>2.29</version> <!-- TODO: Move to 2.31 or later once https://youtrack.jetbrains.com/issue/IDEA-175538 is resolved -->
```

Comment:
please make pom version bumps in a separate PR, since there are more to it than just bumping the versions.
For example in the same swoop I wan't to get rid of all the `@Inject` et.al. stuff. And have this plugin be as much of a model example of how to do pipeline steps as possible.

Reply:
See #36